modules/log/checkHeadings.js

```diff
@@ -0,0 +1,42 @@
+export function checkHeadings(document, filePath, errors) {
+  const headings = Array.from(document.querySelectorAll("h1, h2, h3, h4, h5, h6"));
+  let previousLevel = 0;
+	let h1Count = 0;
+
+	// Check if <h1> is present as the document title and <h2> follows
+	if (!headings.length || headings[0].tagName !== "H1") {
+		if (!errors[filePath]) {
+			errors[filePath] = [];
+		}
+		errors[filePath].push(`Invalid heading structure (document must start with an <h1> heading).`);
```

Comment:
We already have this check in the <title> matches <h1> check. Please see the image below. This may confuse us thinking it's two different errors.

<img width="525" alt="Screenshot 2024-11-01 at 1 11 50 PM" src="https://github.com/user-attachments/assets/aa9d4e99-fda1-4bd6-bfa3-6b7379a78590">

Reply:
I think they are two different errors. We may want to add more details. 

I can change the checkTitleAndH1 file to perhaps be more specific. It can tell us specifically, that the title is missing, or that the title and h1 should match. 

Would that be a better solution than just removing it from the <h1> check?

Reply:
Yes, that works. Thank you.